testhelper/testhelper.go

```diff
@@ -51,14 +54,21 @@ var mockLidarPath = artifact.MustPath("viam-cartographer/mock_lidar")
 
 // SetupStubDeps returns stubbed dependencies based on the camera
 // the stubs fail tests if called.
-func SetupStubDeps(cameraName string, t *testing.T) resource.Dependencies {
+func SetupStubDeps(cameraName, movementSensorName string, t *testing.T) resource.Dependencies {
```

Comment:
I'd leave this as imu. I get why you changed it but since in the future we may allow an odom source to be used as well, or ibstead. I'd say for now keep it as IMU